### DISCUSSION TITLE
gross consumption lower than net?

### DISCUSSION THREAD

**Discussion by stankuhn** [teslamate-org/teslamate#3363] [General]
Hi Guys,

can anyone explain how is it possible that my gross consumption is lower than net consumption?

![image](https://github.com/adriankumpf/teslamate/assets/455893/a8fe7d0f-072a-4eab-ace0-e73620d9405b)

**Comment by ktsenvolt** [comment]
Here is my result for 2020 M3 SR +( 3 yrs old): I don't why you are different.

![image](https://github.com/adriankumpf/teslamate/assets/80360674/405ab469-198d-43bf-8e3f-c9f7f9c16dc1)
